src/injected/styles/injected.scss

```diff
@@ -125,7 +122,7 @@
             fill: $neutral-100 !important;
             fill-opacity: 0.9 !important;
 
-            @media screen and (forced-colors) {
+            @media screen and (forced-colors: active) {
```

Comment:
This (and line 135) omitted the `: active`. It's there in every other case where we check forced-colors, so I made it consistent